modules/mysql/src/main/resources/META-INF/services/org.testcontainers.containers.JdbcDatabaseContainerProvider

```diff
@@ -1 +1 @@
-org.testcontainers.containers.MySQLContainerProvider
\ No newline at end of file
+org.testcontainers.containers.TiDBContainerProvider
```

Comment:
this file belongs to mysql. A new one should be provided in the new `tidb` module

Reply:
My apologies. I changed this file by mistake.